torch/backends/_nnapi/serializer.py

```diff
@@ -469,11 +472,14 @@ def get_tensor_operand_by_jitval_fixed_size(self, jitval):
                 raise Exception("Flexible size is not supported for this operand.")
         return op_id, oper
 
-    def get_tensor_operand_or_constant(self, jitval):
+    def get_tensor_operand_or_constant(self, jitval, dim_order=DimOrder.PRESUMED_CONTIGUOUS):
         operand_id = self.jitval_operand_map.get(jitval)
         if operand_id is None:
             _, value = self.get_constant_value(jitval, "TensorType")
-            operand_id = self.add_tensor_operand_for_weight(value)
+            # assume constants are contiguous
+            if dim_order == DimOrder.CHANNELS_LAST:
+                value = value.permute(0, 2, 3, 1).flatten().reshape(value.shape)
```

Comment:
Is there a better way to do this? If I don't flatten and reshape, broadcasting is a problem with tensor shapes mismatched